lms/services/course.py

```diff
@@ -29,7 +30,7 @@ def any_with_setting(self, group, key, value=True):
             self._db.query(Course)
             .filter(Course.consumer_key == self._consumer_key)
             # pylint: disable=protected-access
-            .filter(Course._settings[group][key] == json.dumps(value))
+            .filter(Course.settings[group][key] == json.dumps(value))
```

Comment:
This `Course.settings[group][key]` syntax is really weird and I wonder what happens if `group` is missing? But :man_shrugging: